direct.mk

```diff
@@ -13,15 +17,22 @@ AUTHORS: .mailmap .git/HEAD
 version/version.go:
 	./version/version.sh > $@
 
-.PHONY: setup
-setup: ## install dependencies
```

Comment:
Any particular reason to rename this? After all, it's not just for CI?

Reply:
It was named "setup" which is way too generic and misleading. What it does is installs some additional software required for CI to run tests. I would not run it anywhere except on a throwaway CI instance, thus the name is chosen to discourage from using it. I mean, I would run `make setup` on my machine, but would take a closer look into `make ci-prep`.